CHANGELOG.adoc

```diff
@@ -17,6 +17,7 @@ Release with new features and bugfixes:
 * https://github.com/devonfw/IDEasy/issues/420[#420]: Add MSI Build and Deployment
 * https://github.com/devonfw/IDEasy/issues/1061[#1061]: Mac: no such file or directory: /_ide/installation/functions
 * https://github.com/devonfw/IDEasy/issues/789[#789]: Add support to uninstall
+* https://github.com/devonfw/IDEasy/issues/654[#654]: Improve plugin support
```

Comment:
Can you please move this to the new release 2025.03.001.

Reply:
Adjusted.